src/main/java/com/microsoft/graph/concurrency/IProgressCallback.java

```diff
@@ -24,14 +24,12 @@
 
 /**
  * A callback that describes how to deal with success, failure, and progress
- * 
- * @param <Result> the result type of the successful action
  */
-public interface IProgressCallback<Result> extends ICallback<Result> {
+public interface IProgressCallback {
```

Comment:
do we still need this since we remove dthe callback?

Reply:
We still need it at one place for upload session to report progress. This way of consumers want to update ui they can